docs/reference/installation.rst

```diff
@@ -5,106 +5,157 @@
 Installation
 ============
 
-* Add ``SonataClassificationBundle`` via composer:
+Prerequisites
+-------------
+
+PHP 5.6 or 7.0 and Symfony 2.8, >=3.3 or 4 are needed to make this bundle work, there are
+also some Sonata dependencies that need to be installed and configured beforehand:
+
+* `SonataEasyExtendsBundle <https://sonata-project.org/bundles/easy-extends>`_
+
+Add ``SonataClassificationBundle`` via composer:
 
 .. code-block:: bash
 
    $ composer require sonata-project/classification-bundle
 
-* Add ``SonataEasyExtendsBundle`` to the dev environment via composer:
+Now, add the new ``SonataClassificationBundle`` to ``bundles.php`` file:
 
-.. code-block:: bash
+.. code-block:: php
+
+    <?php
 
-   $ composer require --dev sonata-project/easy-extends-bundle
+    // config/bundles.php
 
-* Add ``SonataClassificationBundle`` and  ``SonataEasyExtendsBundle`` to your application kernel:
+    return [
+        //...
+        Sonata\ClassificationBundle\SonataClassificationBundle::class => ['all' => true],
+    ];
+
+.. note::
+    If you are not using Symfony Flex, you should enable bundles in your
+    ``AppKernel.php``.
 
 .. code-block:: php
 
+    <?php
+
     // app/AppKernel.php
 
-    public function registerBundles()
+    public function registerbundles()
```

Comment:
this seems wrong, maybe it was right before with B

Reply:
You are right, fixed.